pkg/templates/notificationtemplates/notification_generator.go

```diff
@@ -1 +1,67 @@
 package notificationtemplates
+
+import (
+	"strings"
+
+	"github.com/codeready-toolchain/host-operator/pkg/templates/assets"
+
+	"github.com/pkg/errors"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("notification-templates")
```

Comment:
this is not used anywhere 
```suggestion
```

Reply:
good catch - removed here: 25161ddc4f96a7a916643f77d2c8fdcc908c49e6